### PR TITLE
Further clarified the option of using built-in labels while being able to label non-Office files with the AIP app

### PR DESCRIPTION
**Pull request by E-Saggese**
…e to label non-Office files

The vast majority of customers interviewed didn't know this critical scenario was an option, and as a result most of them were not using built-in labels in Office.

### CODE REVIEW

**Comment by yogkumgit** [comment]
@JohanFreelancer9 - kindly do the copy editor review for this PR. Thanks

File: microsoft-365/compliance/sensitivity-labels-office-apps.md [hunk 1]
```diff
@@ -129,9 +129,10 @@ Deploy this setting by using Group Policy, or by using the [Office cloud policy
 
 ### Office built-in labeling client and the Azure Information Protection client
 
-If users have the [Azure Information Protection client installed](/azure/information-protection/rms-client/aip-clientv2), by default, the built-in labeling client is turned off in their Office apps. 
+You may want to use the built-in sensitivity labeling capabilities in Office to make use of the latest features and enhance user experience, while at the same time having the option of labeling and protecting non-Office files from the desktop with the Azure Information Protection client.
```
Comment (JohanFreelancer9) [inline microsoft-365/compliance/sensitivity-labels-office-apps.md]:
```suggestion
You might want to use the built-in sensitivity labeling capabilities in Office to make use of the latest features and enhance user experience, while at the same time having the option of labeling and protecting non-Office files from the desktop with the Azure Information Protection client. 
```

File: microsoft-365/compliance/sensitivity-labels-office-apps.md [hunk 2]
```diff
@@ -129,9 +129,10 @@ Deploy this setting by using Group Policy, or by using the [Office cloud policy
 
 ### Office built-in labeling client and the Azure Information Protection client
 
-If users have the [Azure Information Protection client installed](/azure/information-protection/rms-client/aip-clientv2), by default, the built-in labeling client is turned off in their Office apps. 
+You may want to use the built-in sensitivity labeling capabilities in Office to make use of the latest features and enhance user experience, while at the same time having the option of labeling and protecting non-Office files from the desktop with the Azure Information Protection client. 
+By default, if users have the [Azure Information Protection client installed](/azure/information-protection/rms-client/aip-clientv2) the built-in labeling client is turned off in their Office apps and the plugin functionality is used instead.
```
Comment (JohanFreelancer9) [inline microsoft-365/compliance/sensitivity-labels-office-apps.md]:
```suggestion
By default, if users have the [Azure Information Protection client installed](/azure/information-protection/rms-client/aip-clientv2), the built-in labeling client is turned off in their Office apps and the plug-in functionality is used instead. 
```

File: microsoft-365/compliance/sensitivity-labels-office-apps.md [hunk 3]
```diff
@@ -129,9 +129,10 @@ Deploy this setting by using Group Policy, or by using the [Office cloud policy
...
+By default, if users have the [Azure Information Protection client installed](/azure/information-protection/rms-client/aip-clientv2) the built-in labeling client is turned off in their Office apps and the plugin functionality is used instead. 
 
-To use built-in labeling rather than the Azure Information Protection client for Office apps, we recommend you use the Group Policy setting **List of managed add-ins** as documented in [No Add-ins loaded due to group policy settings for Office 2013 and Office 2016 programs](https://support.microsoft.com/help/2733070/no-add-ins-loaded-due-to-group-policy-settings-for-office-2013-and-off).
+To maintain the ability to use the Azure Information Protection clientfor labeling non-Office files from the desktop while using the built-in labeling functinality within Office apps, we recommend you use the Group Policy setting **List of managed add-ins** as documented in [No Add-ins loaded due to group policy settings for Office 2013 and Office 2016 programs](https://support.microsoft.com/help/2733070/no-add-ins-loaded-due-to-group-policy-settings-for-office-2013-and-off).
```
Comment (JohanFreelancer9) [inline microsoft-365/compliance/sensitivity-labels-office-apps.md]:
```suggestion
To maintain the ability to use the Azure Information Protection client for labeling non-Office files from the desktop while using the built-in labeling functionality within Office apps, we recommend you use the Group Policy setting **List of managed add-ins** as documented in [No Add-ins loaded due to group policy settings for Office 2013 and Office 2016 programs](https://support.microsoft.com/help/2733070/no-add-ins-loaded-due-to-group-policy-settings-for-office-2013-and-off).
```

File: microsoft-365/compliance/sensitivity-labels-office-apps.md [hunk 4]
```diff
@@ -146,6 +147,14 @@ For Microsoft Word 2016, Excel 2016, PowerPoint 2016, and Outlook 2016, specify
 
 Deploy this setting by using Group Policy, or by using the [Office cloud policy service](/DeployOffice/overview-office-cloud-policy-service).
 
+Alternatively, the following registry value can be used to quickly toggle between using the Azure Information Protection plugin and built-in sensitivity labels in Office apps:
```
Comment (JohanFreelancer9) [inline microsoft-365/compliance/sensitivity-labels-office-apps.md]:
```suggestion
Alternatively, the following registry value can be used to quickly toggle between using the Azure Information Protection plug-in and built-in sensitivity labels in Office apps:
```

File: microsoft-365/compliance/sensitivity-labels-office-apps.md [hunk 5]
```diff
@@ -146,6 +147,14 @@ For Microsoft Word 2016, Excel 2016, PowerPoint 2016, and Outlook 2016, specify
...
+
+|**Registry key**|**Type**|**Value** |
+|:-------|:------:|--------:|
+|HKCU\Software\Microsoft\Office\16.0\Common\Security\Labels\UseOfficeForLabelling  |REG_DWORD  |1  |
+
+Setting this value to 1 enables the built-in sensitivity labeling options in Office, setting it to 0 enables the AIP plugin if it is installed.
```
Comment (JohanFreelancer9) [inline microsoft-365/compliance/sensitivity-labels-office-apps.md]:
```suggestion
Setting this value to **1** enables the built-in sensitivity labeling options in Office; setting it to **0** enables the AIP plug-in if it is installed. 
```

**Review by JohanFreelancer9 — CHANGES_REQUESTED** [review]
Copy review - @E-Saggese, kindly commit the suggested changes. Many thanks.

**Review by JohanFreelancer9 — APPROVED** [review]

**Comment by yogkumgit** [comment]
@cabailey Please merge this PR. Thanks

**Comment by yogkumgit** [comment]
@cabailey Gentle reminder.

**Comment by cabailey** [comment]
Closing after discussion with contributor

**Pull request closed without merging**